tests/reporting/test_resources/markdown_jira_ticket_to_github.md

```diff
@@ -0,0 +1,14 @@
+## 📕 [TECH-290](https://vandebron.atlassian.net/browse/TECH-290) Update PR description in github with deployment info ![sam@vandebron.nl](https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:73eb6738-a8dc-4e71-beb2-16761407e54e/44a3caa2-b498-4ee1-927d-bdb0901a683e/24) 
+Similarly to mpl **modules**, we *should* update the PR details with card description, links to swagger (if they have), links to deployment in cluster, etc.
+
+###### Should support Jira markdown
+
+> Quoted
+
+# first
+# second
+
+[👩‍💻](https://vandebron.atlassian.net/browse/TECH-290/jira/people/6151b89d72f6970069e87968) is tagged
+
+
+🏗️ Build [id](http://localhost/run) ❌ Failed, started by _somebody_
```

Comment:
maybe we should start adding subfolder to the `test_resources`

Reply:
https://github.com/Vandebron/mpyl/pull/84/commits/3782f605bfe48a946a1a45b4520a41db88462b1d